tests/openapi/test_batch_update_multivec.py

```diff
@@ -33,6 +33,39 @@ def assert_points(collection_name, points, nonexisting_ids=None, with_vectors=Fa
         points, key=itemgetter('id')
     )
 
+def test_batch_update_validation(collection_name):
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/batch",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={"operations": [
+            {
+                "upsert": {
+                    "points": [
+                        {
+                            "id": 7,
+                            "vector": {
+                                "image": [1.0, 0.0, 9.0],
+                            },
+                            "payload": {},
+                        },
+                    ],
+                    "update_filter": {
+                        "must": [
+                            {
+                                "key": "empty" # empty field conditions are forbidden
```

Comment:
that's an easy test to validate the missing call

but the entire enum in depth is not validated